otter/test/convergence/test_service.py

```diff
@@ -790,7 +842,7 @@ def plan(*args, **kwargs):
                                status=ScalingGroupStatus.ERROR),
              noop),
             (Log('group-status-error', dict(isError=True, cloud_feed=True,
-                                            status='ERROR')),
+                                            status='ERROR', reasons='')),
```

Comment:
Done! There was some important code being left untested here, especially the `'; '.join` part. Thanks!